changes/31.6.2.md

```diff
@@ -1 +1,11 @@
-- Fix Macedonian Cyrillic Gje under italics (#2493).
+* Fix Macedonian Cyrillic Gje under italics (#2493).
+* Add Characters:
```

Comment:
Please consider bump the version of the change file to 31.7.0 as there's new features added.